content/spin/cache.md

```diff
@@ -0,0 +1,64 @@
+title = "Spin Cache"
+template = "spin_main"
+date = "2022-03-14T00:22:56Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/spin/cache.md"
+
+---
+
+- [The Spin Registry Cache](#the-spin-registry-cache)
+- [Viewing Registry Cache Files](#viewing-registry-cache-files)
+- [Clearing Registry Cache Files](#clearing-registry-cache-files)
+
+## The Spin Registry Cache
+
+A running Spin application can [fetch resources from remote registries](https://developer.fermyon.com/spin/spin-oci) and individual component sources via HTTP endpoints. These resources naturally consume network bandwidth. To ensure network efficiency, and to prevent waiting to download the same files every time an application is started, Spin automatically maintains a local `spin/registry` cache.
+
+## Viewing Registry Cache Files
+
+All downloadable application files that are automatically cached can be found in the following file paths, depending on the operating system being used to run the Spin application:
```

Comment:
Suggestion: before this paragraph, put a note or warning that the registry cache contains no user-serviceable parts; the only safe way to make changes to the contents of the cache is through ~an approved Electrolux dealer~ the Spin command line.  The only operation a user can safely undertake is to delete it.

Reply:
Done, thanks!